### ISSUE TITLE
Uncaught errors print the full source of zstd-codec-binding-wasm

### ISSUE THREAD

**Issue by pimterry** [yoshihitoh/zstd-codec#214]
This is very weird, but 100% reproducible for me on all active Node versions. Any clues welcome!

Repro:
* Create an empty folder
* `npm install zstd-codec`
* Create a `test.js` file containing:
  ```javascript
  const { ZstdCodec } = require('zstd-codec');

  ZstdCodec.run(() => {
      console.log('Zstd loaded');

      setImmediate(() => {
          throw new Error('test error');
      });
  });
  ```
* `node test.js`

This code fails immediately, due to an uncaught exception, but just before it does it prints the entirety of the contents of `zstd-codec-binding-wasm.js` to stderr. It's a huge amount of output, which makes debugging errors (especially failing tests) in any codebase that uses zstd-codec very difficult.

Any idea why this might be happening?

**Comment by pimterry** [comment]
Found it! This is caused by https://github.com/emscripten-core/emscripten/issues/5957

The build output in this module contains:

```javascript
process["on"]("uncaughtException", (function (ex) { if (!(ex instanceof ExitStatus)) { throw ex } }));
process["on"]("unhandledRejection", (function (reason, p) { process["exit"](1) }));
```

This means for everybody using this module in node, all uncaught exceptions get rethrown from inside the emscripten output (creating _enormous_ error output) and all unhandled rejections actively kill the process (which is only the default behaviour in Node 15+).

Libraries like this (i.e. everything except standalone applications) should apparently use `NODEJS_CATCH_EXIT` to disables this behaviour. I'll look at opening a PR to enable that tomorrow.

**Comment by pimterry** [comment]
Oops, my comment auto-closed this! Sorry. This is not actually fixed by #215, but it will be fixed shortly by #216.
